avm/res/compute/image/tests/e2e/max/main.wait.bicep

```diff
@@ -0,0 +1,21 @@
+@description('Optional. The location to deploy to.')
+param resourceLocation string = resourceGroup().location
+
+param serviceShort string
+
+param waitTimeInSeconds string
+
+param tags object
+
+resource deployDelay 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
+  name: '${uniqueString(deployment().name, resourceLocation)}-test-wait-${serviceShort}'
```

Comment:
Any chance this name could be provided via the very top `main.test.bicep` and passed down. When inspecting/troubleshooting the module you'd not only see all relevant names being set in that initial file, you'd also not need to pass top-level information like the `serviceShort`  down.